datanode/gateway/graphql/schema.graphql

```diff
@@ -5969,6 +5985,22 @@ input StopOrderFilter {
   liveOnly: Boolean
 }
 
+input GameTeamScoreFilter {
+  "Zero or more game IDs to filter by"
+  game_ids: [ID!]
```

Comment:
Should this be formatted as ‘gameIds’? Same question for team IDs fields

Reply:
good catch!